pkg/multicluster/local/local_cluster_manager.go

```diff
@@ -8,49 +8,42 @@ import (
 	"fmt"
 
 	"emperror.dev/errors"
-	"fybrik.io/fybrik/manager/apis/app/v1alpha1"
-	"fybrik.io/fybrik/pkg/multicluster"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"fybrik.io/fybrik/manager/apis/app/v1alpha1"
+	"fybrik.io/fybrik/pkg/multicluster"
 )
 
 const (
 	clusterMetadataConfigmapName string = "cluster-metadata"
 )
 
-// ClusterManager for local cluster configuration
-type ClusterManager struct {
+// localClusterManager for local cluster configuration
+type localClusterManager struct {
```

Comment:
In the past we refrained from using names like `localClusterManager` because the name is already in the package name and should not be repeated in the type definition.

Reply:
In general you're right, but here it is a package visibility name, so it will not be used from any other packages with its package name. Therefore golint doesn't complain about it.

When all ClusterManager implementations named ClusterManager it reminds Java obfuscation ;-)